components/spacing.css

```diff
@@ -0,0 +1,281 @@
+/* padding  */
+
+.p-0 {
```

Comment:
Not sure but , padding and margin 0 can go to a reset CSS .

Reply:
Hi! 

Thank you for the review! 

This is not the CSS reset, but the utility classes that let the user set the padding and margin.